src/tools/splash2txt/splash2txt.cpp

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright 2013-2014 Felix Schmitt, Axel Huebl, Rene Widera
+ * Copyright 2013-2015 Felix Schmitt, Axel Huebl, Rene Widera,
+ * Alexander Grund
```

Comment:
Could you please indent your name to align with the first name in the first line?